File: Challenges/Coding World Cup/2023/Day 24 - AUS vs NZ - World Cup Squad Selection.cpp
long long rejection(int n, vector<int> &a, vector<int> &b)
{
    // Write your code here.
    long long ans = 0;
    long long sum = 0;

    for (int i = 0; i < n; i++)
    {
        sum += a[i];
    }

    for (int i = 0; i < n; i++)
    {
        ans = max(ans, sum - a[i] + b[i]);
    }

    return ans;
}

File: Challenges/Coding World Cup/2023/Day 24 - AUS vs NZ - World Cup Squad Selection.java
public class Solution {
    static long rejection(int n, int[] a, int[] b) {
        long ans = 0;
        long sum = 0;

        for (int i = 0; i < n; i++) {
            sum += a[i];
        }

        for (int i = 0; i < n; i++) {
            ans = Math.max(ans, sum - a[i] + b[i]);
        }

        return ans;
    }
}